.gitignore

```diff
@@ -3,6 +3,8 @@
 ##
 ## Get latest from https://github.com/github/gitignore/blob/master/VisualStudio.gitignore
 
+.editorconfig
+
```

Comment:
editorconfig files are frequently version controlled to help enforce consistent style in a team, not a bad idea to just commit this but hey